public/js/app.Ctrl.js

```diff
@@ -0,0 +1,201 @@
+App.controller('App.Ctrl', function($scope, $timeout, AppService) {
+    const colorCode = {1:"green", 2: "red", 3: "yellow", 4: "blue"};
+    const speed = 500; // in milliseconds
+    var on = false;
+    var lock = false;
+    var strict = false;
+    var colorSequence = [];
+    var count = 0;
+    var round = 1;
+    var turnOffTimer;
+    var recursiveTimer;
+    var timer;
+    var sound = {1:{}, 2:{}, 3: {}, 4: {}};
+    var buzzer;
+    var win;
+
+    $scope.init = function() {
+        if(on === true) {
+            setUpSoundEffects();
+            reset(true);
+        }
+    };
+
+    $scope.toggleOnOff = function() {
+        if(on === true) {
+            on = false;
```

Comment:
Definitely remove hard coded css attributes from your controllers.  Angular has a lot of boiler plate code to help you handle display logic.  https://docs.angularjs.org/api/ng/directive/ngClass.  All you need is a scope variable, ie btnOn = false.  Then you leverage angular ngclass expressions, and it will dynamically toggle your css class (always use a class rather than have hard coded css attributes) based on your expression and scope variables.  Try to move all the display logic out of your app.ctrl.  Angular html has access to the scope, you need only initialize your scope variable and let the angular html directives like ngClick change the value of those scope display variables.  In fact, it should only be a small section near the top of app.ctrl where you initialize those scope params with a comment: // initialize scope display variables